tests/kuttl/tests/ovn_db_delete/06-delete-pods-non-graceful.yaml

```diff
@@ -5,3 +5,7 @@ commands:
       oc delete pods -n $NAMESPACE -l service=ovsdbserver-nb --force --grace-period=0
   - script: |
       oc delete pods -n $NAMESPACE -l service=ovsdbserver-sb --force --grace-period=0
+  - script: |
```

Comment:
just remove the test case for force delete, it's invalid.

Reply:
ack, done, thanks for the assist @booxter <3